src/_tutorials/server/get-started.md

```diff
@@ -97,19 +79,16 @@ These commands create a small Dart app that has the following:
   information about which [packages](/guides/packages) the app depends on
   and which versions of those packages are required.
 
-## 5. Get the app's dependencies
-
-Use the [`pub`](/tools/pub/cmd) command to get the packages
-that the app depends on:
+Using `dart create` automatically gets the packages that the app depends on. To do this manually:
 
 ```terminal
-$ pub get
+$ dart pub get
 ```
 
-## 6. Run the app
+## 4. Run the app
 
 To run the app from the command line, use the Dart VM by running the
-[`dart`](/tools/dart-vm) command:
+[`dart`](/tools/dart-run) command:
 
 ```terminal
 $ dart bin/cli.dart
```

Comment:
I am confused between the `dart` and the `dart run` command. What's the difference between these two?

Reply:
I am confused between the `dart` and the `dart run` command. What's the difference between these two?

Reply:
`dart` is the new tool, and it requires a subcommand. It's analogous to the `flutter` tool. Because we already had a tool named `dart` (the VM), we grandfathered in the subcommand-less usage of `dart` for the VM, but we're trying to transition over to the new `dart run`.